SQL/0000-00-00-schema.sql

```diff
@@ -419,6 +419,42 @@ CREATE TABLE `tarchive_files` (
   CONSTRAINT `tarchive_files_TarchiveSeriesID_fk` FOREIGN KEY (`TarchiveSeriesID`) REFERENCES `tarchive_series` (`TarchiveSeriesID`)
 ) ENGINE=InnoDB DEFAULT CHARSET=utf8;
 
+-- ********************************
+-- HRRT PET archive tables
+-- ********************************
+CREATE TABLE `hrrt_archive` (
+  `HrrtArchiveID`     INT(11)          NOT NULL AUTO_INCREMENT,
+  `SessionID`         INT(10) unsigned          DEFAULT NULL,
+  `EcatFileCount`     INT(11)          NOT NULL DEFAULT '0',
+  `NonEcatFileCount`  INT(11)          NOT NULL DEFAULT '0',
+  `DateAcquired`      DATE                      DEFAULT NULL,
+  `DateArchived`      DATETIME                  DEFAULT NULL,
+  `PatientName`       VARCHAR(50)      NOT NULL DEFAULT '',
+  `CenterName`        VARCHAR(50)      NOT NULL DEFAULT '',
```

Comment:
Could it be the CenterID of PSC table?

Reply:
The CenterName field in here refers to the CenterName field read from the HRRT dataset and not to any of the center from the psc table. In the case of the MNI, it will be `BIC-MNI_MCGILL`. Same behaviour as the `tarchive` table.